packages/semver/src/builders/version/schema.d.ts

```diff
@@ -1,15 +1,47 @@
-import { JsonObject } from '@angular-devkit/core';
-
-export interface VersionBuilderSchema extends JsonObject {
-  dryRun?: boolean;
-  noVerify?: boolean;
-  push?: boolean;
-  remote?: string;
-  baseBranch?: string;
-  syncVersions?: boolean;
-  skipRootChangelog?: boolean;
-  skipProjectChangelog?: boolean;
+export type VersioningType = 'independent' | 'sync-group';
+
+export interface SemverConfig {
+  /**
+   * Name of the project or sync-group, for independent versioning this should match the project name.
+   * @example "rx-state"
+   * @example "cdk"
+   */
+  name: string;
+
+  /**
+   * Versioning type.
+   * @values "independent" | "sync-group"
```

Comment:
`sync-group` or `group` ?
I'd rather use `group`. Actually, the type doesn't define the behavior. It will just tell us if this is a single project or a group. If type is `group`, then `packages` option is available.  

By the way, we could use a union of two types `IndependentConfig` & `GroupConfig`.